src/components/Project.js

```diff
@@ -20,20 +20,21 @@ const Project = props => {
             title={name}
           />
           <CardContent>
-            <Typography className="app-title" gutterBottom varient="heading" component="h3">
-              {name}
-            </Typography>
-            <Typography className="description-box" component="p">{description}</Typography>
+            {/* <Typography className="app-title" gutterBottom varient="heading" component="h3">{name}</Typography> */}
```

Comment:
Is this line commented out? If so, do we need it or can we yeet it out?

Reply:
yeet!